Makefile

```diff
@@ -10,3 +10,13 @@ DOCKER_BUILD_CONTEXT = ..
 .PHONY: $(shell test -f common/common.mk || echo >&2 'Please do "git submodule update --init" first.')
 
 include common/common.mk
+
+.PHOHY: generate-new
+generate-new:
+	for version in ${VERSIONS} ; do \
+		if [[ "$$version" == *-minimal ]]; then \
+			python3 ./generator.py -v $$version -m manifest-minimal.yml -s specs/multispec.yml ; \
+		else \
+    		python3 ./generator.py -v $$version -m manifest.yml -s specs/multispec.yml ; \
```

Comment:
This line needs one more indentation.

Reply:
It seems that there is a combination of spaces and tabs on line 20.